File: backend/src/main/java/com/model/office/Office.java
package com.model.office;

import lombok.Data;

import javax.persistence.*;
import java.time.LocalTime;

@Entity
@Table(name="office")
@Data
public class Office {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "office_id")
    private int id;

    @Column(name = "office_city")
    private String city;

    @Column(name = "office_address")
    private String address;

    @Column(name = "office_phone")
    private String phone;

    @Column(name = "office_open")
    private LocalTime openTime;

    @Column(name = "office_close")
    private LocalTime closeTime;

}

File: frontend/src/app/model/office.ts
import {Time} from "@angular/common";

export class Office {
  id: number;
  city: string;
  address: string;
  phone: string;
  openTime: string;
  closeTime: string;


  constructor(office: Office) {
    this.id = office.id;
    this.city = office.city;
    this.address = office.address;
    this.phone = office.phone;
    this.openTime = office.openTime;
    this.closeTime = office.closeTime;
  }

  public toString = () => {
    return this.id + '/' + this.city + '/' + this.address;
  }
}
